pkgs/development/tools/analysis/rr/default.nix

```diff
@@ -51,6 +51,6 @@ stdenv.mkDerivation rec {
 
     license = "custom";
     maintainers = with stdenv.lib.maintainers; [ pierron thoughtpolice ];
-    platforms = ["x86_64-linux"];
+    platforms = stdenv.lib.platforms.x86;
```

Comment:
Are you sure that it actually works? Upstream seems not to build for 32bit: https://github.com/mozilla/rr/releases

Reply:
For one release they said:

```
 No 32-bit release packages included because my 32-bit VM broke. Let's see if anyone complains.
```

so it could still work.

Reply:
Hmm.

*looks*

hahaha: from the 5.1.0 release notes:

> No 32-bit release packages included because my 32-bit VM broke. Let's see if anyone complains.

Reply:
And just tried it with a 32bit xterm + bash and ... seems to work? Not obviously broken anyway :D.

Reply:
This will probably bit-rot over time, but then hopefully nobody will care about 32-bit.